critiquebrainz/ws/review/views.py

```diff
@@ -233,19 +235,36 @@ def review_modify_handler(review_id, user):
     """
 
     def fetch_params():
-        text = Parser.string('json', 'text', min=REVIEW_MIN_LENGTH, max=REVIEW_MAX_LENGTH)
-        return text
+        try:
+            text = Parser.string('json', 'text', min=REVIEW_TEXT_MIN_LENGTH, max=REVIEW_TEXT_MAX_LENGTH)
+        except MissingDataError:
+            text = 'same'                        # Assign temporary value which indicates no modification
```

Comment:
This doesn't seem right. What if the review text is actually `same`, then the comparisons afterwards are going to break. Sure, it doesn't match the constraints for the text, so it's currently not possible to write a review like that. But this might change in the future and in any case that seems very hacky way of doing things.

Reply:
Agreed. This is not subtle. I added some changes which are much cleaner than this. Don't know why I couldn't think of such a simple solution earlier.